File: Modul-4/Soal-3/PRAK403-2410817210027-HAFIZPERDANA.py
angka1, angka2 = map(int,input("").split())

i = angka1
j = angka2

if i < j:
    while i<=angka2 and j>=angka1:
        print(f"{i} {j}", end=" ")

        if i < angka2 or j > angka1:
            print("-", end=" ")
        i +=1
        j -=1
    
elif i > j:
    while i>=angka2 and j<=angka1:
        print(f"{i} {j}", end=" ")

        if i > angka2 or j < angka1:
            print("-", end=" ")
        i -=1
        j +=1

File: Modul-4/Soal-3/PRAK403-2410817210027-HAFIZPERDANA.c
#include <stdio.h>

int main ()
{
    int angka1, angka2;

    scanf("%d %d", &angka1, &angka2);

    for (int i=angka1, j=angka2; i<=angka2 && j>=angka1; i++, j--) {
        printf("%d %d", i, j);
        if (i<angka2 || j>angka1) {
            printf(" - ");
        }
    }

    for (int i=angka1, j=angka2; i>=angka2 && j<=angka1; i--, j++) {
        printf("%d %d", i, j);
        if (i>angka2 || j<angka1) {
            printf(" - ");
        }
    }

    return 0;
} 